packages/code/src/main.rs

```diff
@@ -36,11 +36,14 @@ fn manifest() {
                         https://github.com/sublimehq/Packages"},
                     {"name": "font_size", "default": 12, "description": "The size of the font", "type": "uint"},
                     {"name": "tab_size", "default": 4, "description": "The size tabs will be adjusted to", "type": "uint"},
-                    {"name": "theme", "default": "mocha", "description":
+                    {"name": "theme", "default": "default", "type": ["default", "ocean-dark", "ocean-light", "mocha", "eighties", "github", "solar-dark", "solar-light"], "description":
                         "Theme of the code section. For available themes, see \
                         https://docs.rs/syntect/latest/syntect/highlighting/struct.ThemeSet.html#method.load_defaults"},
                     {"name": "bg", "default": "default", "description": "Background of the code section"},
                 ],
+                "variables": {
+                    "code_theme": {"type": "const", "access": "read"}
```

Comment:
Maybe document somewhere (like on the theme argument, or transform descr) that this is used?

Reply:
This issue still remain